src/LRS.js

```diff
@@ -747,7 +747,12 @@ TinCan client library
 
             for (i = 0; i < idProps.length; i += 1) {
                 if (typeof cfg.params[idProps[i]] !== "undefined") {
-                    params[idProps[i]] = cfg.params[idProps[i]].id;
+                    if (typeof cfg.params[idProps[i]] === "string") {
+                        params[idProps[i]] = cfg.params[idProps[i]];
+                    }
+                    else {
+                        params[idProps[i]] = cfg.params[idProps[i]].id;
+                    }
```

Comment:
I think it makes sense to allow any object with an "id" property to work because that would have always worked and we don't want to break backwards compat, it would have been better to require a Verb or Activity at the beginning, but oh well.